microsoft-edge/devtools-guide-chromium/performance/reference.md

```diff
@@ -295,70 +570,187 @@ DevTools assigns scripts random colors.  In the previous figure, function reques
 
 If you want to hide the detailed flame chart of JavaScript requests, see [Disable JavaScript samples](#disable-javascript-samples), above.
 When JavaScript samples are disabled, only high-level events are displayed, such as `Event: input` and `Function Call` from the previous figure.
-<!--When JS samples are disabled, you only see high-level events such as `Event (click)` and `Function Call` (script_foot_closure.js:53) from Figure 16.-->
+<!--When JS samples are disabled, you only see high-level events such as `Event (click)` and `Function Call` (script_foot_closure.js:53). -->
+
+
+<!-- ------------------------------ -->
+#### Read the flame chart
+<!-- https://developer.chrome.com/docs/devtools/performance/reference#flame-chart -->
+
+The **Performance** tool represents main thread activity in a flame chart.  The x-axis represents the recording over time.  The y-axis represents the call stack.  The events on top cause the events below.
+
+<!-- ![A flame chart](todo: flame-chart.png) -->
```

Comment:
14x:   `](todo: ` possibly out of scope for PR.  Scenario: copied heading & text from upstream, didn't create png yet.

Reply:
status: copied refreshed.png as flame-chart.png, but might not fit w/ the following text from upstream

Reply:
result:
https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/perf-sync/microsoft-edge/devtools-guide-chromium/performance/reference.md#read-the-flame-chart

might enter comment about the commented-out text from upstream that might not match refreshed.png that's here renamed as flame-chart.png